### PR TITLE
Hybridize

### PR DESCRIPTION
**Pull request by danyakarras**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,12 +9,36 @@
     "package.json"
   ],
   "dependencies": {
-    "polymer": "^1.9.1",
-    "d2l-colors": "^2.4.0",
+    "polymer": "Polymer/polymer#^1.9.1 || ^2.0.0",
+    "d2l-colors": "^2.4.0 || ^3.0.0",
```
Comment (dlockhart) [inline bower.json]: I'd go with colors `^3.1.0`, it's the latest fanciness

File: bower.json [hunk 2]
```diff
@@ -9,12 +9,36 @@
...
-    "web-component-tester": "^6.0.0"
+    "d2l-demo-template": "^1.0.0",
+    "web-component-tester": "^6.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0 || ^1.0.0"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.0.0",
+        "d2l-colors": "^2.2.3",
```
Comment (dlockhart) [inline bower.json]: And go with `^2.4.0` here

File: bower.json [hunk 3]
```diff
@@ -9,12 +9,36 @@
...
   "devDependencies": {
-    "web-component-tester": "^6.0.0"
+    "d2l-demo-template": "^1.0.0",
+    "web-component-tester": "^6.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0 || ^1.0.0"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.0.0",
```
Comment (dlockhart) [inline bower.json]: And `^1.9.1` here

**Comment by dlockhart** [comment]
I'd recommend telling the Polymer linter that you'd like it to lint for hybrid mode by adding a `polymer.json` file, like [this one](https://github.com/BrightspaceUI/checkbox/blob/master/polymer.json).

File: bower.json (hunk 1 above)
Comment (danyakarras) [inline bower.json]: fancy? ok :)

File: bower.json (hunk 2 above)
Comment (danyakarras) [inline bower.json]: kk

File: bower.json (hunk 3 above)
Comment (danyakarras) [inline bower.json]: gotcha

**Comment by danyakarras** [comment]
oh right! I forgot that step. Thanks!

**Comment by danyakarras** [comment]
:D

**Pull request merged**